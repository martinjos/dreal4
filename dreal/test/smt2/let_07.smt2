(set-logic QF_NRA)
(declare-const a Real)
(assert (= a 2))
(assert
  (let ((b (* a 4))
        (c 6))
    (= (* b c) 48)))
(check-sat)
(exit)
