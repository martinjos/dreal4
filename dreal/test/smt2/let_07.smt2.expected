delta-sat with delta = 0.001
